From a29b1a579f016b1256597a47152d9ee544231e99 Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Sat, 30 Nov 2024 07:43:57 -0500
Subject: [PATCH 3/4] Rework aidl call, add aidl call to agold hal

---
 sepolicy/treble_app.te | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/sepolicy/treble_app.te b/sepolicy/treble_app.te
index 6a6bdb1..4bed5f1 100644
--- a/sepolicy/treble_app.te
+++ b/sepolicy/treble_app.te
@@ -35,6 +35,7 @@ allow system_app mtk_vilte_support_prop:property_service set;
 type hal_gesturewake_hwservice, hwservice_manager_type;
 type hal_gesturewake_default, domain;
 allow system_app hal_gesturewake_hwservice:hwservice_manager { find };
-allow system_app hal_gesturewake_default:binder { call transfer };
-allow hal_gesturewake_default system_app:binder { call transfer };
+binder_call(system_app, hal_gesturewake_default);
 
+type hal_aidl_agolddaemon_default, domain;
+binder_call(system_app, hal_aidl_agolddaemon_default);
-- 
2.34.1

